frontend/src/static/js/v_summary.js

```diff
@@ -518,8 +536,10 @@ window.vSummary = {
 
     // updating filters programically //
     resetDateRange() {
-      this.tmpFilterSinceDate = this.minDate;
-      this.tmpFilterUntilDate = this.maxDate;
+      // set both dates to empty string which will call their respective
```

Comment:
A suggestion from me:
The main purpose of this should be `let their watcher remove the hash` right? You only mentioned `what to do` but not mention `why do so`. Usually, we are able to read `what to do` from the code directly, but we may not understand `why do so`.

Reply:
Yes the main purpose of this is to `let watcher remove the hash` and `let watcher reset both tmpFilterSinceDate and tmpFilterUntilDate to minDate and maxDate respectively`. OK will amend the comment. Thanks for the feedback.